preprocess_image.py

```diff
@@ -157,6 +159,13 @@ def __call__(self, image):
     normal[~mask] = 0
     del dpt_normal_model
 
+    if opt.dont_recenter_resize:
+        # write output
+        cv2.imwrite(out_rgba, cv2.cvtColor(carved_image, cv2.COLOR_RGBA2BGRA))
```

Comment:
Please don't duplicate code like this, it might get out-of-date with the version below... so best to surround the next part with `if opt.recenter_resize` or something like that.

Reply:
Done!